conf/default.toml

```diff
@@ -36,17 +36,12 @@ background_color = "#fdfbff"
 # [translations.fr]
 # "results.sortBy.dateStamp" = "Dernière fois que quelqu'un a modifié quelque chose"
```

Comment:
Shouldn't this example be without quotes? Did I understand right, that quotes are only used for custom translations that display quotes in the app text?

Reply:
> Did I understand right, that quotes are only used for custom translations that display quotes in the app text?

Not sure what you mean:
* quotes around the key (left part) are optional (they weren't before this PR)
* quotes around the translated text (right) are mandatory since TOML requires strings to be quoted

Reply:
Ok, got it